src/functions.php

```diff
@@ -841,3 +841,15 @@ function fingerprint(callable $callback, string $fingerprint): void
         }
     }
 }
+
+/**
+ * Check if a command exists. This function is not reliable on Windows.
```

Comment:
It has to run on windows

Reply:
Yes, I haven't looked at how to check on Windows yet, but since the docker part is going to be part of a second PR I'll see this point when the time comes. I'm deleting it for now because it's not for cases other than docker.